transformation/src/clients/adapter-client.ts

```diff
@@ -0,0 +1,13 @@
+import axios from 'axios'
+
+const ADAPTER_SERVICE_URL = process.env.ADAPTER_SERVICE_URL || 'http://localhost:8082'
```

Comment:
Hm that means the transformation service knows the adapter service.. Does it make sense to pass the whole URL so that we avoid this coupling?